python/tank/deploy/tank_commands/clone_configuration.py

```diff
@@ -90,6 +90,8 @@ def run_noninteractive(self, log, parameters):
                          computed_params["path_mac"], 
                          computed_params["path_win"])
 
+        self.tk.log_metric("clone config")
```

Comment:
Awesome, this is what I was looking for. I know we talked about this, but I was looking in the Action base class. Thanks for pointing these out!  I've made these changes.